src/components/CustomerQuota/ItemsSelection/ItemsSelectionCard.tsx

```diff
@@ -31,7 +40,40 @@ export const ItemsSelectionCard: FunctionComponent<ItemsSelectionCard> = ({
   updateCart
 }) => {
   const [isAddUserModalVisible, setIsAddUserModalVisible] = useState(false);
-  const { getFeatures, products } = useProductContext();
+
+  const { getFeatures, products, features } = useProductContext();
+  const { showAlert } = useContext(AlertModalContext);
+
+  const onModalCheck = async (input: string): Promise<void> => {
```

Comment:
Actually what does `onModalCheck` mean ah?

Reply:
ah `onModalCheck` is called when `onCheck` is triggered in `AddUserModal` (the child component). it handles the logic that was in `onCheck` and controls `AddUserModal`, as well as any error modals that may be displayed in the process. 
we moved the logic to the parent because there are bugs when multiple modals are displayed simultaneously, making it impossible to show the error modals in `AddUserModal`.